generator/src/main/java/com/graphicsfuzz/generator/util/RemoveImmediateContinueStatements.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 The GraphicsFuzz Project Authors
+ * Copyright 2019 The GraphicsFuzz Project Authors
```

Comment:
Nit: I would leave the copyright year as 2018.

Reply:
Fixed.